ui/src/views/NotFoundPage/NotFoundPage.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <cv-content id="#main-content">
+    <div class="page__details">
```

Comment:
Could you add the AppHeader here:
```vue
  <cv-content id="#main-content">
    <AppHeader />
```

I think the page looks a bit less stark with the header.